app/src/main/res/drawable/spotlight_overlay_hint_background.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+  android:shape="rectangle">
+  <corners
+    android:bottomLeftRadius="5dp"
```

Comment:
Ideally these values should be `4dp` but if still its not possible for your case then I think it can be an exception.